prototype2/common/TimeString.cpp

```diff
@@ -0,0 +1,16 @@
+/** Copyright (C) 2017 European Spallation Source ERIC */
+
+#include <common/TimeString.h>
+
+std::string timeString() {
+  char cStartTime[50];
+  time_t rawtime;
+  struct tm *timeinfo;
+  time(&rawtime);
+  timeinfo = localtime(&rawtime);
+  strftime(cStartTime, 50, "%Y%m%d-%H%M%S", timeinfo);
+  std::string startTime = cStartTime;
```

Comment:
It is possible to remove a few lines of code here. I will leave it to you to decide to do it or not.

Reply:
leaving this as is until we get the `date` library working